docs/source/examples/substrafl/get_started/run_mnist_torch.ipynb

```diff
@@ -249,16 +249,16 @@
         "import numpy as np\n",
```

Comment:
<p>we create 2 tasks per orga (first bullet point)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>